clio-client/src/test/scala/org/broadinstitute/clio/client/dispatch/AddExecutorSpec.scala

```diff
@@ -90,21 +80,23 @@ class AddExecutorSpec extends BaseClientSpec with AsyncMockFactory {
     val otherMetadata = ArraysMetadata(chipType = Some("the-type"))
 
     val ioUtil = mock[IoUtil]
-    (ioUtil.readFileData _)
-      .expects(loc)
-      .returning(otherMetadata.asJson.pretty(defaultPrinter))
+    (ioUtil
+      .readMetadata(_: ArraysIndex.type)(_: URI))
+      .expects(*, loc)
+      .returning(Source.single(otherMetadata))
 
     val executor = new AddExecutor(AddWgsUbam(theKey, loc, false))
-    recoverToSucceededIf[IllegalArgumentException] {
+    recoverToSucceededIf[Exception] {
```

Comment:
Why? The more-specific exception is useful for making sure we fail in the expected way.